packages/shared/src/common/date.ts

```diff
@@ -65,7 +65,17 @@ export function elapsedTimeFromNow(
 }
 
 export function buildDayjs(date?: ConfigType, format?: string, strict?: boolean): dayjs.Dayjs {
-  return dayjs.utc(date, format, strict);
+  const dayjsObj = dayjs.utc(date, format, strict);
+  
+  // Only check for years less than 1900 if we have a valid date
```

Comment:
No need for comments unless we're explaining "why" we're doing something that's not obvious by reading the code - [reference](https://www.notion.so/metriport/Development-Best-Practices-bb3574d3c83d46dc9914a9e5c5542ca7?pvs=4#fb560de23853473f88ea7011630eb6b0)